2.4.1/bullseye/include/entrypoint

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+set -e
+
+if [[ $UID == "${ASTRONOMER_UID:-1000}" ]]; then
+  # Since we need to support running tini as another user, we can't put tini in
+  # the ENTRYPOINT command, we have to run it here, if we haven't already
+  if [[ -z "$__TINIFIED" ]]; then
+    __TINIFIED=1 exec tini -- "$0" "$@"
+  fi
+else
+  __TINIFIED=1 exec gosu "${ASTRONOMER_USER}" tini -- "$0" "$@"
+fi
+
+if [[ -n "$EXECUTOR" && -z "$AIRFLOW__CORE__EXECUTOR" ]]; then
+  # Support for puckle style of defining configs
```

Comment:
If "puckle style" isn't a typo, I'm curious what it is.

Reply:
https://github.com/puckel/docker-airflow
and
https://hub.docker.com/r/puckel/docker-airflow

It should be "puckel" I guess.

Reply:
At one point in time it was a much more common thing in the community.

Reply:
TIL, thanks!